### PR TITLE
Add tweak to hide trending badges

### PR DESCRIPTION
**Pull request by ThePsionic**
img
Those things.
(also yay three 3's in the version)

### CODE REVIEW

File: Extensions/tweaks.js [hunk 1]
```diff
@@ -139,6 +139,11 @@ XKit.extensions.tweaks = new Object({
 			default: false,
 			value: false
 		},
+		"hide_trending": {
+			text: "Hides \"Trending\" badges at the bottom of posts",
```
Comment (nightpool) [inline Extensions/tweaks.js]: In the future you can use single quotes to avoid escaping double quotes.

**Comment by nightpool** [comment]
lgtm @homu r+

**Comment by dlmarquis** [comment]
I made this change silently back when the New XKit project first started. It's the hide_explore option. Now we've got duplicate options.

**Comment by ThePsionic** [comment]
@dlmarquis whelp.

**Pull request merged**